misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-metasfresh/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+
+metasfresh.mass.json.request.directory.path=./src/main/resources/mass-process
```

Comment:
how is this going to work? can it be overwritten via environment variables?
at any rate, we generally use a "data" folder when we deploy this in docker

Reply:
yes, it can be overwritten